package.json

```diff
@@ -8,7 +8,8 @@
   "files": [
     "cjs",
     "esm",
-    "umd"
+		"umd",
+		"index.d.ts"
```

Comment:
You seem to be using tabs, but the rest of the file doesn't.

Reply:
Oops, my bad